modules/hooks/activeEffect.js

```diff
@@ -0,0 +1,13 @@
+import AreaHelpers from "../system/area-helpers.js";
+
+export default function() {
+    Hooks.on("createActiveEffect", async (effect, data, user) => {
+        if(game.user.isUniqueGM && effect.flags?.wfrp4e?.applicationData?.type) {
+            if (effect.flags.wfrp4e.applicationData.type == "aura" && (effect.flags.wfrp4e.applicationData.targetedAura == "target" || effect.flags.wfrp4e.applicationData.targetedAura == "all") || 
+                effect.flags.wfrp4e.applicationData.type == "area" &&
+                game.canvas?.scene) {
+                AreaHelpers.checkAreas(game.canvas.scene);
+            }
+        }
+    });
+}
```

Comment:
For one thing, where possible, I am avoiding the use of hooks and trying to consolidate it within the CRUD functions of the document itself. 

Secondly, this should already be handled by `_onCreateDescendantDocuments` in the actor.

Reply:
removed